adobe/react-spectrum
# fix(#4049): Express theme should render a border on Popovers and Dialogs with forced-colors: active

Closes #4049

## ✅ Pull Request Checklist:

- [x] Included link to corresponding [React Spectrum GitHub Issue #4049](https://github.com/adobe/react-spectrum/issue/4049).

## 📝 Test Instructions:

1. In Chrome, open [ActionMenu story using Express theme](https://reactspectrum.blob.core.windows.net/reactspectrum/5fda2564952f6d294b1c2c72ab0ea6eecd07b3bf/storybook/index.html?path=/story/actionmenu--default&providerSwitcher-express=true)
2. In Chrome DevTools, under Rendering > Emulate CSS media feature forced-colors, select `forced-colors: active`.
3. Confirm that when expanded, the ActionMenu popover has a 1px border using the foreground color for `forced-colors: active`.
4. Repeat for other components that open a dropdown menu or listbox, such as [ComboBox](https://reactspectrum.blob.core.windows.net/reactspectrum/5fda2564952f6d294b1c2c72ab0ea6eecd07b3bf/storybook/index.html?path=/story/combobox--default&providerSwitcher-express=true), [DatePicker](https://reactspectrum.blob.core.windows.net/reactspectrum/5fda2564952f6d294b1c2c72ab0ea6eecd07b3bf/storybook/index.html?path=/story/date-and-time-datepicker--default&providerSwitcher-express=true), [DateRangePicker](https://reactspectrum.blob.core.windows.net/reactspectrum/5fda2564952f6d294b1c2c72ab0ea6eecd07b3bf/storybook/index.html?path=/story/date-and-time-daterangepicker--default&providerSwitcher-express=true), [MenuTrigger](https://reactspectrum.blob.core.windows.net/reactspectrum/5fda2564952f6d294b1c2c72ab0ea6eecd07b3bf/storybook/index.html?path=/story/menutrigger--default-menu-static&providerSwitcher-express=true), [Picker](https://reactspectrum.blob.core.windows.net/reactspectrum/5fda2564952f6d294b1c2c72ab0ea6eecd07b3bf/storybook/index.html?path=/story/picker--default), and [SearchAutocomplete](https://reactspectrum.blob.core.windows.net/reactspectrum/5fda2564952f6d294b1c2c72ab0ea6eecd07b3bf/storybook/index.html?path=/story/searchautocomplete--static).
5. Open and test each of the Dialog variants, starting with [Dialog -- Default](https://reactspectrum.blob.core.windows.net/reactspectrum/5fda2564952f6d294b1c2c72ab0ea6eecd07b3bf/storybook/index.html?path=/story/dialog--default&providerSwitcher-express=true), to verify that Dialogs have a border and border-radius. 
6. Full Screen Takeover stories, [Dialog - fullscreenTakeover form](https://reactspectrum.blob.core.windows.net/reactspectrum/5fda2564952f6d294b1c2c72ab0ea6eecd07b3bf/storybook/index.html?path=/story/dialog--fullscreentakeover-form&providerSwitcher-express=true), [DialogContainer - type: fullscreenTakeover](https://reactspectrum.blob.core.windows.net/reactspectrum/5fda2564952f6d294b1c2c72ab0ea6eecd07b3bf/storybook/index.html?path=/story/dialogcontainer--type-fullscreentakeover&providerSwitcher-express=true), or [DialogTrigger - type: fullscreenTakeover](https://reactspectrum.blob.core.windows.net/reactspectrum/5fda2564952f6d294b1c2c72ab0ea6eecd07b3bf/storybook/index.html?path=/story/dialogtrigger--type-fullscreentakeover&providerSwitcher-express=true), do not have a border or border-radius on the fullscreenTakeover dialog.

## 🧢 Your Project:

Adobe/Accessibility
